wagtail/images/static_src/wagtailimages/js/hallo-plugins/hallo-wagtailimage.js

```diff
@@ -15,7 +15,7 @@
                     uuid: this.options.uuid,
                     editable: this.options.editable,
                     label: 'Images',
-                    icon: 'icon-picture',
+                    icon: 'icon-image',
```

Comment:
Does changing the icon name not cause problems when integrators re-use icons for their own blocks or menu items?

Reply:
Ah, yes. I’ve been careful to change as little as possible, so the only change to names is that:

- `icon-picture` is removed. Use `icon-image` instead.
- `icon-file-text-alt` is removed. Use `icon-doc-full` instead.

Both of these were only used in Hallo as far as I can see – I’ll add a note in the release notes.